lhotse/recipes/xbmu_amdo31.py

```diff
@@ -0,0 +1,126 @@
+"""
+About the XBMU-AMDO31 corpus
+XBMU-AMDO31 is an open-source Amdo Tibetan speech corpus published by Northwest Minzu University.
+publicly available on https://huggingface.co/datasets/syzym/xbmu_amdo31
+"""
+
+import logging
+import os
+import shutil
+import tarfile
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, Optional, Union
+
+from tqdm.auto import tqdm
+
+from lhotse import validate_recordings_and_supervisions
+from lhotse.audio import Recording, RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, safe_extract, urlretrieve_progress
+
+
+def download_xbmu_amdo31(
+    target_dir: Pathlike = ".",
+) -> Path:
+    """
+    Downdload and untar the dataset
+    :param target_dir: Pathlike, the path of the dir to storage the dataset.
+    :param force_download: Bool, if True, download the tars no matter if the tars exist.
+    :param base_url: str, the url of the OpenSLR resources.
+    :return: the path to downloaded and extracted directory with data.
+    """
+    target_dir = Path(target_dir)
```

Comment:
It seems here you are assuming that a tar.gz file is already downloaded in the `target_dir`? If so, please add some message here in case these are not present. For example, does the user need to download these files manually?